packages/nextjs-kit/package.json

```diff
@@ -99,7 +99,7 @@
 		"vitest": "^0.34.5"
 	},
 	"peerDependencies": {
-		"next": "^13.1.5",
+		"next": "13.5.6",
```

Comment:
It's OK to keep the `^` here, otherwise only exactly `v13.5.6` of next will work.